content/content.types.ts

```diff
@@ -122,3 +122,9 @@ export interface BenefitItem {
     leftOffset: number;
     rightOffset: number;
 }
+
+export interface PerfectFitCard {
+    id: string;
+    header: string;
+    body:string;
```

Comment:
Can you add a space after the colon here.

Reply:
ooff one day I'll properly space my interfaces, fixed